projects/observability/src/shared/components/card-list/card-list.component.scss

```diff
@@ -1,5 +1,9 @@
 @import 'mixins';
 
+@mixin grouped-style($color: white) {
+  box-shadow: 0 3px 0 -1px $color, 0 4px 0 -1px $gray-2;
```

Comment:
This alone doesn't add the grouping, does it?

Reply:
nope. To keep things simple, grouping is controlled by the card content. Here we are just setting style.

Reply:
Not sure I follow - isn't grouping purely a style change? We wouldn't want every type of card content to have to implement grouped visualization on its own.